### PR TITLE
EE-1158: add dev & test keycloak urls, use dev as default

### PR DESCRIPTION
**Pull request by mtCarto**
[EE-1158](https://bcmines.atlassian.net/browse/EE-1158)

### CODE REVIEW

**Comment by BcGovNeal** [comment]
This change will require the same the same changes on the API.  Otherwise there will be a mismatch in the issuer.

I think it would be cleaner to use the config service to set the Keycloak host rather than parsing the URL.

**Comment by KitArmstrong** [comment]
> ~This change will require the same the same changes on the API. Otherwise there will be a mismatch in the issuer.~ API can be configured through `SSO_ISSUER` and `SSO_JWKSURI` envs.
> 
> I think it would be cleaner to use the config service to set the Keycloak host rather than parsing the URL.

I think that is a great idea and a good use case for the service.

File: src/app/services/keycloak.service.ts [hunk 1]
```diff
@@ -15,25 +15,29 @@ export class KeycloakService {
...
-      // Always enable sso
-      // case 'http://localhost:4200':
-      //   // Local
-      //   this.keycloakEnabled = false;
-      //   break;
-
-      case 'https://eagle-dev.pathfinder.gov.bc.ca':
-        // Dev etc
-        this.keycloakEnabled = true;
+      case 'http://esm-prod.pathfinder.bcgov/':
```
Comment (MaxWardle) [inline src/app/services/keycloak.service.ts:23]: This link doesn't work for me

**Comment by mtCarto** [comment]
Agreed, will be updating now that config service is ready.

**Pull request closed without merging**